src/library_glfw.js

```diff
@@ -1006,13 +1006,63 @@ var LibraryGLFW = {
       for (i = 0; i < GLFW.windows.length && GLFW.windows[i] !== null; i++) {
         // no-op
       }
-      if (i > 0) throw "glfwCreateWindow only supports one window at time currently";
 
       // id for window
       id = i + 1;
 
       // not valid
       if (width <= 0 || height <= 0) return 0;
+      var canvasID = 'canvas' + (id - 1);
+      var ctxID = 'ctx' + (id - 1);
+
+      if (!document.getElementById('canvas')) {
+        mainCanvas = document.createElement('canvas');
+        mainCanvas.id = 'canvas';
+        mainCanvas.width = width;
+        mainCanvas.height = height;
+        Module['canvas'] = mainCanvas;
```

Comment:
It looks like ` Module.canvas` is used elsewhere in this file (without the quotes).

Reply:
Should i change them all to Module.canvas?

Reply:
It looks like the quoted version is correct, at least according to the existing code in this file.

Reply:
Alright. Other way round.